correct the description of Supermarket's search endpoint (#1886)
chef/chef-web-docs: chef_master/source/supermarket_api.rst

/search
-----------------------------------------------------

> Search returns cookbooks that match a query. The query syntax is a simple, fuzzy text search on cookbook names, cookbook descriptions, and the cookbook owners' usernames.

Reviewer: @robbkidd How's this for word-smithing? 
```suggestion
Search performs a fuzzy, keyword search on cookbook names, cookbook descriptions, and the cookbook owners' usernames.
```

Author: Those are some well-smithed words. Smelted down and hammered into brevity. 👍
